packages/app/src/debug/DebugCancelledAlert.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <Alert
+    :title="t('debugPage.manuallyCancelled')"
+    status="warning"
+    :icon="ErrorOutlineIcon"
+    class="flex flex-col mb-24px w-full"
+  >
+    <div class="flex items-center">
+      <div>{{ t('debugPage.specsSkipped', {n: totalSpecs, totalSkippedSpecs}) }}</div>
+      <template v-if="cancellation.cancelledBy?.email && cancellation.cancelledBy.fullName">
+        <div class="rounded-full font-semibold bg-orange-500 h-3px mx-10px w-3px" />
+        <div class="flex items-center">
+          <UserAvatar
+            :email="cancellation.cancelledBy.email"
+            class="h-20px mr-7px w-20px"
+            data-cy="cancelled-by-user-avatar"
+          />
+          <div>{{ cancellation.cancelledBy.fullName }}</div>
+        </div>
+      </template>
+      <template v-if="cancellation.cancelledAt">
+        <div class="rounded-full font-semibold bg-orange-500 h-3px mx-10px w-3px" />
+        <div>
+          {{ dayjs(cancellation.cancelledAt).utc().format('MMM D, YYYY h:mm A UTC') }}
```

Comment:
I wrote a big comment about this (please double check my logic/reasoning).

Assuming `cancelledAt` is in UTC (from what you said, it is) this should be:

```suggestion
          {{ dayjs(cancellation.cancelledAt).local().format('MMM D, YYYY h:mm A') }}
```

Reply:
updated with https://github.com/cypress-io/cypress/pull/25286/commits/0bf96fec361908af7619a5e28f61486cb4d0d0c7 👍🏻